example/package.json

```diff
@@ -14,9 +14,9 @@
     "start": "electron ."
   },
   "dependencies": {
-    "about-window": "*"
+    "about-window": "1.4.0"
```

Comment:
Is there any reason to fix version?  If fixing it, we need to take care about it when bumping up version.